damnit/ctxsupport/ctxrunner.py

```diff
@@ -304,9 +315,14 @@ def execute(self, run_data, run_number, proposal, input_vars) -> 'Results':
                     # Dependency within the context file
                     if annotation.startswith("var#"):
                         dep_name = annotation.removeprefix("var#")
-                        if dep_name in dep_results:
+                        match = fnmatch.filter(dep_results, dep_name)
+
+                        if len(match) == 1 and match[0] == dep_name:
                             dep_data = dep_results[dep_name]
                             kwargs[arg_name] = dep_data
+                        elif len(match) > 1:
```

Comment:
So a wildcard dependency implies we'll skip running the downstream variable, it will never be passed an empty dict? I'm OK with that, as you can always give it a default `{}` if you want it to run with no matches.